scheduleit/views/meetings/show.php

```diff
@@ -11,7 +14,7 @@
     $attendeeOnids = $_POST['attendeeOnid'];
     $link = $_POST['link'];
     $link = $_SERVER['HTTP_ORIGIN'] . $link;
```

Comment:
In my last PR, I changed the `site_url` in twig to be a full path, so I don't think you need to use `$_SERVER['HTTP_ORIGIN']` anymore.